versions/development/SPEC.md

```diff
@@ -3054,15 +2869,14 @@ Array[Pair[Int, String]] zipped = zip(xs, ys)     # i.e.  zipped = [ (1, "a"), (
 
 ## Array[Pair[X,Y]] cross(Array[X], Array[Y])
 
-Given any two Object types, the `cross` function returns the cross product of those Object types in the form of a Pair object.
+Return the cross product of the two arrays. Array[Y][1] appears before
```

Comment:
Typo: should be `Array[1][Y]`? 
(But TBH this sentence could probably do with some revising)